blog/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class User(AbstractUser):
+    def __str__(self):
+        return self.username
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        to=User,
+        on_delete=models.CASCADE,
+        related_name="posts",
+    )
+    title = models.CharField(max_length=255)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        verbose_name = "Post"
+        verbose_name_plural = "Posts"
+
+    def __str__(self):
+        return self.title
+
+
+class Commentary(models.Model):
+    user = models.ForeignKey(
+        to=User,
+        on_delete=models.CASCADE,
+        related_name="commentarys",
+    )
+    post = models.ForeignKey(
+        to=Post,
+        on_delete=models.CASCADE,
+        related_name="commentarys"
+    )
+    created_time = models.DateTimeField(auto_now_add=True)
+    content = models.TextField()
+
+    class Meta:
+        ordering = ["-created_time"]
+        verbose_name = "commentary"
+        verbose_name_plural = "commentarys"
```

Comment:
The `verbose_name_plural` for `Commentary` should be 'commentaries' instead of 'commentarys'. This correction ensures consistency with standard English pluralization.

Reply:
i want write it like "commentarys", it is my idea